main/performance.h

```diff
@@ -106,6 +106,26 @@ class Performance : public Object {
 		PIPELINE_COMPILATIONS_SURFACE,
 		PIPELINE_COMPILATIONS_DRAW,
 		PIPELINE_COMPILATIONS_SPECIALIZATION,
+		NAVIGATION_2D_ACTIVE_MAPS,
```

Comment:
As mentioned in the server enum we might want to reconsider the debug monitors if we are changing things now.